docs/static/upgrading.asciidoc

```diff
@@ -66,37 +124,42 @@ Keep in mind that only one Logstash instance can write to `path.queue`. You
+supported. We recommend that you upgrade to 6.x, and then upgrade to 7.x.
+
 ==== When to Upgrade
 
 Fresh installations can and should start with the same version across the Elastic Stack.
 
-Elasticsearch 5.0 does not require Logstash 5.0. An Elasticsearch 5.0 cluster will happily receive data from a
-Logstash 2.x instance via the default HTTP communication layer. This provides some flexibility to decide when to upgrade
-Logstash relative to an Elasticsearch upgrade. It may or may not be convenient for you to upgrade them together, and it
-is
-not required to be done at the same time as long as Elasticsearch is upgraded first.
+Elasticsearch 7.0 does not require Logstash 7.0. An Elasticsearch 7.0 cluster
+will happily receive data from earlier versions of Logstash via the default
+HTTP communication layer. This provides some flexibility to decide when to
+upgrade Logstash relative to an Elasticsearch upgrade. It may or may not be
+convenient for you to upgrade them together, and it is not required to be done
+at the same time as long as Elasticsearch is upgraded first.
 
-You should upgrade in a timely manner to get the performance improvements that come with Logstash 5.0, but do so in
+You should upgrade in a timely manner to get the performance improvements that come with Logstash 7.0, but do so in
 the way that makes the most sense for your environment.
 
 ==== When Not to Upgrade
 
-If any Logstash plugin that you require is not compatible with Logstash 5.0, then you should wait until it is ready
+If any Logstash plugin that you require is not compatible with Logstash 7.0, then you should wait until it is ready
 before upgrading.
 
-Although we make great efforts to ensure compatibility, Logstash 5.0 is not completely backwards compatible. As noted
-in the Elastic Stack upgrade guide, Logstash 5.0 should not be upgraded before Elasticsearch 5.0. This is both
-practical and because some Logstash 5.0 plugins may attempt to use features of Elasticsearch 5.0 that did not exist
-in earlier versions. For example, if you attempt to send the 5.x template to a cluster before Elasticsearch 5.0, then it
-will not be able to use it and all indexing will fail likely fail. If you use your own, custom template with Logstash,
+Although we make great efforts to ensure compatibility, Logstash 7.0 is not completely backwards compatible. As noted
+in the Elastic Stack upgrade guide, Logstash 7.0 should not be upgraded before Elasticsearch 7.0. This is both
+practical and because some Logstash 7.0 plugins may attempt to use features of Elasticsearch 7.0 that did not exist
+in earlier versions. For example, if you attempt to send the 7.x template to a cluster before Elasticsearch 7.0, then 
+all indexing will fail likely fail. If you use your own custom template with Logstash,
 then this issue can be ignored.
 
 Note the Elasticsearch Output Index Template change in the <<breaking-changes>> documentation for further insight into
```

Comment:
At line 165, does the note about Output Index Template breaking change come out for 7.0?

Reply:
I suppose, if the user upgraded via 6.7 then they might have read this Note